### PR TITLE
Do not adopt copied CSVs

### PR DESCRIPTION
**Pull request by ecordell**
**Description of the change:**
- Disowns any adopted copied CSVs if they exist in the cluster
- Prevents adoption of copied CSVs in the future

**Motivation for the change:**
Copied CSVs can show up in the component list of an Operator if:

- Component adoption happens before the copy
- A namespace is added to the operatorgroup's set after the real csv has been adopted

**Reviewer Checklist**
- [ ] Implementation matches the proposed design, or proposal is updated to match implementation
- [ ] Sufficient unit test coverage
- [ ] Sufficient end-to-end test coverage
- [ ] Docs updated or added to `/doc`
- [ ] Commit messages sensible and descriptive

### CODE REVIEW

File: pkg/controller/operators/olm/operatorgroup_test.go [hunk 1]
```diff
@@ -121,6 +121,65 @@ func TestCopyToNamespace(t *testing.T) {
...
+					Namespace: "bar",
+					Labels: map[string]string{
+						"operators.coreos.com/foo": "",
+						"operators.coreos.com/bar": "",
+						"untouched": "fine",
+					},
+				},
+				Status: v1alpha1.ClusterServiceVersionStatus{
+					Message:        "The operator is running in foo but is managing this namespace",
+					Reason:         v1alpha1.CSVReasonCopied},
```
Comment (hasbro17) [inline pkg/controller/operators/olm/operatorgroup_test.go:150]: Nit:
```suggestion
					Reason:         v1alpha1.CSVReasonCopied
					},
```

**Review by hasbro17 — APPROVED** [review]
/lgtm

**Comment by timflannagan** [comment]
```
INFO[2021-04-08T17:06:16Z] time="2021-04-08T17:05:12Z" level=fatal msg="failed to listen: listen tcp 127.0.0.1:50054: bind: address already in use" 
INFO[2021-04-08T17:06:16Z] FAIL	github.com/operator-framework/operator-lifecycle-manager/pkg/package-server/provider	0.096s 
```

/retest

**Pull request merged**